ts-tests/tests/runtime-upgrade.test.ts

```diff
@@ -0,0 +1,71 @@
+import { blake2AsHex, cryptoWaitReady } from '@polkadot/util-crypto';
+import * as fs from 'fs';
+import { Keyring, ApiPromise, WsProvider } from '@polkadot/api';
+import { loadConfig, describeLitentry } from './utils';
+import '@polkadot/wasm-crypto/initOnlyAsm';
+import * as path from 'path';
+import { expect } from 'chai';
+import { step } from 'mocha-steps';
+
+async function getRuntimeVersion(api: ApiPromise) {
+    const runtime_version = await api.rpc.state.getRuntimeVersion();
+    return +runtime_version['specVersion'];
+}
+
+async function runtimeUpgrade(api: ApiPromise, wasm: string) {
+    const keyring = new Keyring({ type: 'sr25519' });
+    const alice = keyring.addFromUri('//Alice');
+    const old_runtime_version = await getRuntimeVersion(api);
+    let currentBlock = (await api.rpc.chain.getHeader()).number.toNumber();
+    console.log(`Start doing runtime upgrade, current block = ${currentBlock}`);
+
+    // authorize and enact the upgrade
+    let nonce = (await api.rpc.system.accountNextIndex(alice.address)).toNumber();
+    await api.tx.sudo
+        .sudo(api.tx.parachainSystem.authorizeUpgrade(blake2AsHex(wasm)))
+        .signAndSend(alice, { nonce: -1 });
+    console.log('Submitted authorizeUpgrade');
+    await api.tx.parachainSystem.enactAuthorizedUpgrade(wasm).signAndSend(alice, { nonce: -1 });
+    console.log('Submitted enactAuthorizedUpgrade');
+
+    // wait for 10 blocks
+    console.log('Wait for new runtime ...');
+    currentBlock = (await api.rpc.chain.getHeader()).number.toNumber();
+    let timeoutBlock = currentBlock + 10;
+    let runtimeUpgraded = false;
+
+    return new Promise(async (resolve, reject) => {
```

Comment:
Although the whole return value here is a little bit hard to understand, it works fine.

Reply:
https://github.com/litentry/litentry-parachain/blob/d714d251661e8bb3c534ca6827c90c44fef44790/ts-tests/tests/runtime-upgrade.test.ts#L37-L61